common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala

```diff
@@ -22,41 +22,85 @@ object PhotographersList {
 
 object MetadataConfig {
   val staffPhotographers = Map(
+    // Current
+    "David Sillitoe" -> "The Guardian",
+    "Graham Turner"  -> "The Guardian",
+    "Sean Smith"     -> "The Guardian",
+    "Jill Mead"      -> "The Guardian",
+    "Roger Tooth"    -> "The Guardian",
+    "James Mann"     -> "The Guardian",
+    "Rachel Vere"    -> "The Guardian",
+    "Alicia Canter"  -> "The Guardian",
+    "Bill Code"      -> "The Guardian",
+
+    // Past
```

Comment:
Jo B has assured me that these people shouldn't come in through the FTP stuff, so we should be safe.

The alternative would be to move this list to the Picdar import, but then when things are uploaded, we might not get the benefit of adding this automagically.

:roller_coaster: